projects/packages/connection/src/class-rest-connector.php

```diff
@@ -216,6 +226,65 @@ public static function activate_plugins_permission_check() {
 		return new WP_Error( 'invalid_user_permission_activate_plugins', self::get_user_permissions_error_msg(), array( 'status' => rest_authorization_required_code() ) );
 	}
 
+	/**
+	 * Permission check for the connection_plugins endpoint
+	 *
+	 * @return bool|WP_Error
+	 */
+	public static function connection_plugins_permission_check() {
+		if ( true === static::activate_plugins_permission_check() ) {
+			return true;
+		}
+
+		if ( true === static::is_request_signed_by_jetpack_debugger() ) {
+			return true;
+		}
+
+		return new WP_Error( 'invalid_user_permission_activate_plugins', self::get_user_permissions_error_msg(), array( 'status' => rest_authorization_required_code() ) );
+
+	}
+
+	/**
+	 * Verifies if the request was signed with the Jetpack Debugger key
+	 *
+	 * @return bool
+	 */
+	public static function is_request_signed_by_jetpack_debugger() {
```

Comment:
Do you think it would be valuable to add unit tests for `is_request_signed_by_jetpack_debugger`?

Reply:
sure. will do.

Reply:
done!